src/main/java/uk/gov/companieshouse/company/metrics/exception/ControllerExceptionHandler.java

```diff
@@ -46,9 +57,43 @@ public ResponseEntity<Object> handleException(Exception ex, WebRequest request)
         responseBody.put("correlationId", correlationId);
         request.setAttribute("javax.servlet.error.exception", ex, 0);
         logger.error("correlationId = " + correlationId, ex);
+
         return new ResponseEntity<>(responseBody, HttpStatus.INTERNAL_SERVER_ERROR);
     }
 
+    @ExceptionHandler(value = {DataAccessResourceFailureException.class})
+    public ResponseEntity<Object> handleException(DataAccessResourceFailureException ex, WebRequest request) {
```

Comment:
[non-blocking] Seems like the request argument in all of these handlers is redundant

Reply:
Missed request.setAttribute("javax.servlet.error.exception", ex, 0);